lms/templates/calculator/toggle_calculator.html

```diff
@@ -22,9 +22,9 @@
           <ul id="calculator_input_help" class="help" aria-hidden="true">
             <li class="hint-item" id="hint-moreinfo">
                 <p><span class="bold">${_("For detailed information, see {math_link_start}Entering Mathematical and Scientific Expressions{math_link_end} in the {guide_link_start}edX Guide for Students{guide_link_end}.").format(
-                  math_link_start='<a href="http://edx-guide-for-students.readthedocs.org/en/latest/SFD_mathformatting.html">',
+                  math_link_start='<a href="http://edx.readthedocs.io/projects/edx-guide-for-students/en/latest/completing_assignments/SFD_mathformatting.html#math-formatting">',
```

Comment:
Oh, sorry! The answer is that Read the Docs provides alternate URLs for each subproject. Each of our documents is a subproject of the "edx" project. The URL for a subproject can either be structured with the subproject name as a subdomain or in the resource path under the main project domain. These are equivalent:

```
http://edx.readthedocs.io/projects/edx-guide-for-students/
http://edx-guide-for-students.readthedocs.io/

```
So the URLs in the code here use inconsistent syntax but they both work as expected and I chose to make fewer changes.

Thanks for reviewing!